terraform/cloud.gov.tf

```diff
@@ -1,19 +1,16 @@
-variable "cloudfoundry_elb_logging_development" {
-  default = "dualstack.development-CloudFoundry-Logging-1588361105.us-gov-west-1.elb.amazonaws.com"
-}
+locals {
+  cloud_gov_cloudfront = "d2vy872d33xc5d.cloudfront.net"
+  cloud_gov_green_elb = "dualstack.cf-green-elb-1b9dvw4o0ubi0-1450531403.us-east-1.elb.amazonaws.com."
 
-variable "cloudfoundry_elb_logging_staging" {
-  default = "dualstack.staging-CloudFoundry-Logging-538826588.us-gov-west-1.elb.amazonaws.com"
-}
-variable "cloudfoundry_elb_logging_production" {
-  default = "dualstack.production-CloudFoundry-Logging-910586631.us-gov-west-1.elb.amazonaws.com"
-}
+  cloudfoundry_elb_logging_development = "dualstack.development-CloudFoundry-Logging-1588361105.us-gov-west-1.elb.amazonaws.com"
+  cloudfoundry_elb_logging_staging = "dualstack.staging-CloudFoundry-Logging-538826588.us-gov-west-1.elb.amazonaws.com"
+  cloudfoundry_elb_logging_production = "dualstack.production-CloudFoundry-Logging-910586631.us-gov-west-1.elb.amazonaws.com"
 
-variable "elb_prometheus_staging" {
-  default = "dualstack.staging-Prometheus-658384006.us-gov-west-1.elb.amazonaws.com"
-}
-variable "elb_prometheus_production" {
-  default = "dualstack.production-Prometheus-1971082399.us-gov-west-1.elb.amazonaws.com"
+  elb_prometheus_staging = "dualstack.staging-Prometheus-658384006.us-gov-west-1.elb.amazonaws.com"
+  elb_prometheus_production = "dualstack.production-Prometheus-1971082399.us-gov-west-1.elb.amazonaws.com"
+
+  cloud_gov_cloudfront_zone_id = "Z2FDTNDATAQYW2"
+  cloud_gov_elb_zone_id = "Z35SXDOTRQ7X7K"
```

Comment:
This is the zone id for elb in us-east-1: http://docs.aws.amazon.com/general/latest/gr/rande.html#elb_region. Also not specifically related to cloud.gov afaict.